### DISCUSSION TITLE
Why has *nx workspaces* been removed from the docs?

### DISCUSSION THREAD

**Discussion by vidrepar** [ben-rogerson/twin.macro#580] [Q&A] [answered]
Nx workspaces have been listed in the docs, but they are not anymore.

See: https://github.com/ben-rogerson/twin.macro/commit/70a772626a3e02d4771efe50b92240b091c4aee1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L222

What was the reason for the removal?

**Comment by ben-rogerson** [comment, marked as answer]
There are a couple of pull requests to add an twin + nx example but I felt it was a step up in file/setup complexity compared with the existing examples.
Keen to hear any thoughts.
